plugin/src/leiningen/figwheel.clj

```diff
@@ -56,16 +56,20 @@
         (catch Exception e#
           (do
             (.printStackTrace e#)
-           (System/exit 1))))
+            (System/exit 1))))
      requires)))
 
 (defn run-compiler [project {:keys [all-builds build-ids] :as autobuild-opts}]
-  (run-local-project
-   project all-builds
-   '(require 'figwheel-sidecar.repl-api)
-   `(do
-      (figwheel-sidecar.repl-api/system-asserts)
-      (figwheel-sidecar.repl-api/start-figwheel-from-lein '~autobuild-opts))))
+  (if (empty? all-builds)
```

Comment:
This can occur if someone sets `:load-all-builds` to `false` but forgets to list any builds on the command. Figwheel exits anyway in that case, I just added a warning message so the user knows why.